Electric-Phantoms/welcome-to-nashville
# Sh display park itinerary

# Description

* limited number of items showing in display to 10


Fixes # (issue)

## Type of change

Please delete options that are not relevant.


# Testing Instructions

- git fetch --all
- git checkout sh-display-park-itinerary
- choose "tennis courts"
- verify only 10 items are displayed
- select Save button
- verify displays on itinerary
- choose "basketball courts"
- verify previous items are replaced by only 10 new items
- select Save button
- verify displays on itinerary


# Checklist:
